packages/angular-display/README.md

```diff
@@ -13,3 +13,23 @@ yarn add @cloudflight/angular-display
 # or
 pnpm add @cloudflight/angular-display
 ```
+
+## Configuration
+
+To use the library in a typesafe way you need to extend the `Breakpoints` interface using declaration merging.
+The interface is exposed in the global namespace `ClfIsDisplay`.
+
+Add the following to a `.d.ts` file that is picked up by the compiler:
+
+```ts
+declare namespace ClfIsDisplay {
```

Comment:
Why `namespace` and not the module name?

Reply:
I initially tried it with the module name, but with that the `ValidOptions` type used for the directive and pipe input did not update correctly. Thats when I found the namespace option that is also used by [Express](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/express-serve-static-core/index.d.ts) as documented at the namespace declaration.